modules/game.rb

```diff
@@ -5,13 +5,13 @@ class Game < Item
   def initialize(publish_date, multiplayer, last_played_at)
     super(publish_date)
     @multiplayer = multiplayer
-    @last_played_at = Date.parse(last_played_at)
+    @last_played_at = last_played_at
   end
 
   private
 
   def can_be_archived?
-    if Time.now.year - @last_played_at.year > 2 && super
+    if Time.now.year - Date.parse(@last_played_at).year > 2 && super
```

Comment:
- [optional] It makes your code more readable if you put the subtracting part inside parentheses like:
`(Time.now.year - Date.parse(@last_played_at).year) > 2`

Reply:
@mohisa302  corrected based on comments